### DISCUSSION TITLE
"Seed Fix" mod is causing a problem but there is no seed fix mod in the mods folder??

### DISCUSSION THREAD

**Discussion by CoreOptd** [FabricMC/fabric-api#2047] [Player support]
Idk if this is the right place to ask this but I'm having a problem when I try to open my fabric mod pack but it gives an error in which it says:
```
Incompatible mod set!
net.fabricmc.loader.impl.FormattedException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'Seed Fix' (seedfix) 1.0.1 with any version that is compatible with:
		 - minecraft 1.18.1
Unmet dependency listing:
	 - Mod 'Seed Fix' (seedfix) 1.0.1 requires version 1.18.2-rc.1 or later of 'Minecraft' (minecraft), but only the wrong version is present: 1.18.1!
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:190)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:71)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```
From what I understood it says that I have a newer version of Seed Fix installed, but here is the thing I don't have a mod that is called Seed Fix in my mods folder (or anything that resembles it). Screenshot of my mods folder is attached to this post.
![modsfolder](https://user-images.githubusercontent.com/95182007/156929343-f86f27d1-ea43-4edf-a2a7-85cc8e9b45e5.png)

**Comment by JFronny** [comment]
Fabric mods can include dependencies. In your case, starmutes mods, specifically terralith, include seedfix. Use the correct version of terralith for your minecraft version.

**Reply by CoreOptd** [reply]
Thank you so much!

**Reply by J0eBidenn** [reply]
I am having the same problem here, would you please tell which mods may be the ones causing the problem? 
![image](https://user-images.githubusercontent.com/101452466/158011875-0c6a6932-1970-4d6e-9465-5dd41ee36ba4.png)
![image](https://user-images.githubusercontent.com/101452466/158011885-be4891f2-35b0-4305-a6b0-67f116f74b97.png)
![image](https://user-images.githubusercontent.com/101452466/158011939-5b0f88df-cc20-4c9d-9540-fcd8218c2757.png)
![image](https://user-images.githubusercontent.com/101452466/158011964-e232f549-87ef-4c68-a07e-693bba898117.png)
![image](https://user-images.githubusercontent.com/101452466/158011984-d9a65aab-9353-4228-b08d-60493a7eb76a.png)
oh, and yes, i am aware that is is an unhealthily huge amount of mods.

**Reply by J0eBidenn** [reply]
ohh ok never mind i updated terralith and it worked.
